src/test/ui/check_match/issue-43253.stderr

```diff
@@ -15,10 +15,22 @@ warning: unreachable pattern
    |
 43 |         8...9 => {},
    |         ^^^^^
+   |
+note: lint level defined here
+  --> $DIR/issue-43253.rs:12:9
+   |
+12 | #![warn(unreachable_patterns)]
+   |         ^^^^^^^^^^^^^^^^^^^^
 
 warning: unreachable pattern
   --> $DIR/issue-43253.rs:49:9
    |
 49 |         9...9 => {},
    |         ^^^^^
+   |
+note: lint level defined here
```

Comment:
This looks like a regression in one-time diagnostics (the [motivation for which was avoiding too many "lint level defined here"s](https://github.com/rust-lang/rust/issues/24690)), but it's not immediately clear why that would happen ...

Reply:
I think this is just a mixup that happened due to the update-all script not actually updating everything after it produced the expected output once. I need to `touch` all files in order to have them regenerate their stderr files.